src/coreComponents/unitTests/constitutiveTests/testPVT_data/deadOilPVDO.txt

```diff
@@ -0,0 +1,21 @@
+1.000e+07 1.23331 1.5674e-04
```

Comment:
Also not for this PR, but we have lots of PVDO/PVDG tables floating around in various places.  Would be nice to organize them somehow into a common, well-labeled directory of default examples.

Reply:
Agree. We have some in the unit test directory. However they are not well formed (the monotonicity of Bg for example isn't as expected) and GEOS gives warnings.